cmd/unused-exporter/exporter.go

```diff
@@ -272,3 +233,48 @@ func (e *exporter) Collect(ch chan<- prometheus.Metric) {
 		}
 	}
 }
+
+func getDiskLabels(d unused.Disk, v bool) []any {
+	diskLabels := []any{
+		slog.String("name", d.Name()),
+		slog.Int("size_gb", d.SizeGB()),
+		slog.Time("created", d.CreatedAt()),
+	}
+
+	if v {
+		meta := d.Meta()
+		diskMetaLabels := make([]any, 0, len(meta))
+		for _, k := range meta.Keys() {
+			diskMetaLabels = append(diskMetaLabels, slog.String(k, meta[k]))
+		}
+		diskLabels = append(diskLabels, diskMetaLabels...)
+	}
+
+	return diskLabels
+}
+
+func getNamespace(d unused.Disk, p unused.Provider) string {
+	if strings.ToLower(p.Name()) == "azure" {
+		return d.Meta()["kubernetes.io-created-for-pvc-namespace"]
+	}
+
+	return d.Meta()["kubernetes.io/created-for/pvc/namespace"]
+}
+
+func addMetric(ms []metric, p unused.Provider, d *prometheus.Desc, v float64, lbls ...string) []metric {
+	return append(ms, metric{
+		desc:   d,
+		value:  v,
+		labels: append([]string{strings.ToLower(p.Name()), p.ID()}, lbls...),
+	})
+}
+
+func lastUsedTS(d unused.Disk) (float64) {
+	var ts float64
+	lastUsed := d.LastUsedAt()
+	if !lastUsed.IsZero() {
+		ts = float64(lastUsed.UnixMilli())
+	}
+
+	return ts
+}
```

Comment:
Couple of _Goisms_ here, no need to change, tho.
```suggestion
func lastUsedTS(d unused.Disk) float64 {
	lastUsed := d.LastUsedAt()
	if lastUsed.IsZero() {
		return 0
	}

	return float64(lastUsed.UnixMilli())
}
```